### PR TITLE
Add paint example

### PR DESCRIPTION
**Pull request by afdw**

### CODE REVIEW

File: examples/paint/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        .set_property("border", "solid")
+        .unwrap();
+    let context = canvas
+        .get_context("2d")
+        .unwrap()
+        .unwrap()
+        .dyn_into::<web_sys::CanvasRenderingContext2d>()
+        .unwrap();
+    let context = Rc::new(context);
+    let pressed = Rc::new(RefCell::new(false));
```
Comment (CryZe) [inline examples/paint/src/lib.rs]: Instead of RefCell you can use just a normal Cell

**Comment by alexcrichton** [comment]
Awesome, thanks!

**Comment by fitzgen** [comment]
@afdw, want to add a subsection to the guide that shows off this example? :)

See `guide/src/web-sys/examples/*`

**Comment by afdw** [comment]
@fitzgen https://github.com/rustwasm/wasm-bindgen/pull/823

**Pull request merged**